T04.01

Innborne opened the pull request:
-added styled-components module
-used styled-components for read-only mode style
-small changes in styles for cards checkboxes

inavumau committed:
added styled-components module

inavumau committed:
used styled-components for read-only mode style
-small changes in styles for cards checkboxes

kkaroukinIBA commented on src/components/Cards/CardsItem.css:
```diff
@@ -48,7 +48,7 @@
 }
 
-.cards-item-checked {
+.cards-item.checked {
```
i think .checked should be enough

kkaroukinIBA commented on src/components/Cards/CardsItem.js:
```diff
@@ -99,10 +99,8 @@ function CardsItem({ card, onSaveCardData, readOnly }) {
-
   return (
-    <div className={`cards-item ${cardChecked}`}>
+    <div className={`cards-item ${checkedValue ? "checked" : ""}`}>
```
can you please for these cases use classnames library

kkaroukinIBA commented on src/components/Cards/CardsMain.js:
```diff
@@ -15,15 +30,14 @@ function CardsMain(props) {
-          className="cards-read-checkbox"
           checked={readOnlyMode}
           onChange={readOnlyHandleClick}
         ></input>
```
Empty tags should look like <Tag />

kkaroukinIBA commented on src/components/Cards/CardsMain.js:
```diff
@@ -1,7 +1,22 @@
+import styled from "styled-components";
 import "./CardsMain.css";
 
+const CheckboxContainer = styled.div`
```
It shouldn't be wrapper for input. It should be input to be used.

inavumau committed:
added classnames library

inavumau committed:
switched to classnames library for cards styles

inavumau committed:
added CardsCheckbox component
-used styled-components for CardsCheckbox
-read-only mode checkbox switched to CardsCheckbox component
